.ci/pitest.sh

```diff
@@ -209,7 +209,7 @@ pitest-coding)
   "MultipleVariableDeclarationsCheck.java.html:<td class='covered'><pre><span  class='survived'>            if (newNode.getLineNo() &#62; currentNode.getLineNo()</span></pre></td></tr>"
   "MultipleVariableDeclarationsCheck.java.html:<td class='covered'><pre><span  class='survived'>                || newNode.getLineNo() == currentNode.getLineNo()</span></pre></td></tr>"
   "RequireThisCheck.java.html:<td class='covered'><pre><span  class='survived'>                    &#38;&#38; ast1.getColumnNo() &#60; ast2.getColumnNo()) {</span></pre></td></tr>"
-  "RequireThisCheck.java.html:<td class='covered'><pre><span  class='survived'>        final boolean methodNameInMethodCall = parentType == TokenTypes.DOT</span></pre></td></tr>"
+  "RequireThisCheck.java.html:<td class='covered'><pre><span  class='survived'>        return parentType == TokenTypes.DOT</span></pre></td></tr>"
```

Comment:
as we touching this code ...
we need to resolve(cover it) this survival.

They existed only as technical debt, as we touch we should resolve it.

Reply:
It survives being changes to "always true." I tried removing the whole method, and all the tests still passed, because at least one other bug (in my opinion) still makes the tests pass. As it currently is, the expression/method the surviving code is part of just makes the check realize that there is no violation _faster_.

It could be removed, but then non-violations would be incorrectly found, and not reported only because `getClassFrameWhereViolationIsFound` fails to produce a frame for them.

I find it very strange that `getClassFrameWhereViolationIsFound` also removes false positives, and I think I fixed at least part of that behavior, but I intend to create a new ticket for that, because some tests validate bugs. E.g. `InputRequireThisEnumInnerClassesAndBugs` contains the line
```
return b + b * b;
```
and test expects only `174:16` and `174:20` to marked as violations, the first two `b` occurrences. So I intend it separately that the 3rd `b` in this example, among others, should be reported, and that may make it possible for mutations to kill the expression being discussed. Other than that I have no idea what token could replace the "dot" that would currently cause a test failure. Someone else is welcome to come up with that idea. Or would it be fine for me to add a whole new set of changes to this one that deals with the false negatives?